backend/User/views.py

```diff
@@ -146,8 +147,6 @@ def get(self, request, user_id):
 				image_data = image_file.read()
 				encoded_image = base64.b64encode(image_data).decode('utf-8')
 				return Response({'image': encoded_image}, status=status.HTTP_200_OK)
-		except ProfilePicture.DoesNotExist:
```

Comment:
Whats up? We throw an error if the photo is not found or does the encoded picture just return as empty image?

Reply:
so if the image_data is not there and we feed null to the base64 encoding, its fine, even with the decode question? to me this change looks to be unnecessary.

Reply:
Maybe just make it explicit that we return empty image + status 200 thingie if profile picture is not found, thanks!